torch_xla/csrc/tensor.cpp

```diff
@@ -1422,14 +1450,15 @@ std::shared_ptr<XLATensor::Async> XLATensor::ScheduleSyncTensorsGraph(
       }
     } catch (...) {
       // There are two paths of discovery of an exception happening on an
-      // asynchronous task. One happens if the creator of the asynchronous task
-      // explicitly waits for completion, in which case the exception will be
-      // thrown from the Wait() API. Re-throwing the exception below makes sure
-      // this will be captured by the completer function created below, and
-      // surfaced by the Wait() API. But we also need to surface the exception
-      // even in case the caller does not wait, and that is accomplished by
-      // setting the unlockers status. In that case the exception will be
-      // surfaced when the user tries to acquire the device locks the next time.
+      // asynchronous task. One happens if the creator of the asynchronous
+      // task explicitly waits for completion, in which case the exception
+      // will be thrown from the Wait() API. Re-throwing the exception below
+      // makes sure this will be captured by the completer function created
+      // below, and surfaced by the Wait() API. But we also need to surface
+      // the exception even in case the caller does not wait, and that is
+      // accomplished by setting the unlockers status. In that case the
+      // exception will be surfaced when the user tries to acquire the device
+      // locks the next time.
```

Comment:
Can you undo these unintentional comment format change to make this pr a bit cleaner :D?